saleor/dashboard/category/forms.py

```diff
@@ -19,6 +19,19 @@ def __init__(self, *args, **kwargs):
     class Meta:
         model = Category
         exclude = ['slug']
+        labels = {
+            'name': pgettext_lazy(
+                'Category field',
+                'category name'),
```

Comment:
It would be enough to omit ```category``` and leave just ```name``` as a message, as it was originally. Same below.